docs/about-lightning.md

```diff
@@ -0,0 +1,228 @@
+scalable and secure way.
+
+##### ...but providing a fully open source web app with a user-friendly web interface.
+
+A fully open source web app, it can be deployed anywhere through Docker and
+comes with a user-friendly, low-code interface with the full functionality
+needed for organizations to build, run and audit their workflows all in one
+place. ![Lightning](/img/lightning_build_run_audit.png)
+
+##### Build
+
+Empower more users in your organization to have a say in what gets automated and
+how. Lightning’s visual interface makes workflows more intelligible to
+non-technical users, bridging the gap between the IT specialists that build out
+automations and program managers that are the real business/ program experts on
+the processes that need automating.
+
+##### Audit
+
+Treat every workflow run with the care and attention it deserves. In OpenFn,
+each incoming request or transaction that gets processed is more than a piece of
+data - it represents a vulnerable child in need of critical support, a farmer
+managing their savings to make sure they can afford the next harvest. Lightning
+provides users with a dashboard that allows them to monitor the health of their
+integrations to make sure no request goes unprocessed.
+
+## What is Lightning ?
+
+[OpenFn/Lightning](https://github.com/OpenFn/lightning/) is a _fully open
+source_ workflow automation platform. It is designed for governments or NGOs who
+need a flexible solution which enables them to integrate and connect _any
+system_ and comes with fully-fledged user management and auditing capabilities.
+Lightning relies on the same tried-and-trusted core technology as the OpenFn
+iPaaS and comes with an improved, visual interface for building integrations.
+
+:::note
+
+Any integration built on the OpenFn iPaaS is can be run on Lightning.
+
+:::
+
+## Guiding principles
+
+Lightning is developed in line with the
+[principles for digital development](https://digitalprinciples.org/principles/)
+and under the guidance of it's Open Source Steering Committee which you can read
+about [here](https://openfn.github.io/governance/OSSC.html).
+
+On top of this, Lightning follows 4 key principles which determine how it should
+be developed:
+
+### Integrations break
+
+Anyone that has worked on integration projects in the past is well aware that
+integrations do break. No matter how well designed they are, the fact is that
+**they connect multiple systems that all change over time**: new API versions
+get released, data models change, IDs, codes and mappings change, data standards
+are updated and the processes themselves evolve. This is why Lightning will
+include:
+
+Enhanced testing and debugging:
+
+- Save inputs as edge cases for robust workflow testing
+- View the input and output for each step in a workflow to easily identify where
+  an error occurred
+- Throw custom errors to improve API messages (adaptors)
+- Add custom logic to handle a step failure (fail triggers)
```

Comment:
Change `step` to `workflow step` or `process step`? (Otherwise with little context, people might not know what a `step` is